spanner/spec/spec_helper.rb

```diff
@@ -33,8 +33,19 @@
     @restored_database_id = "restored_db_#{seed}"
     @spanner              = Google::Cloud::Spanner.new project: @project_id
     @instance             = @spanner.instance @instance_id
+    @mr_instance_id       = "rb-test-#{seed}"
+    # Create multi-region instance for CMEK
+    create_instance @project_id, @mr_instance_id, "nam3"
```

Comment:
This line is erroring now in the tests, because the arguments are supposed to be keyword arguments rather than positional.